cupy/cuda/stream.py

```diff
@@ -22,7 +22,7 @@ class Event(object):
 
     """
     def __init__(self, block=False, disable_timing=False, interprocess=False):
-        self.ptr = None
+        self.ptr = 0
```

Comment:
`self.ptr` is set at the last line of `__init__`.  So I guess it is not necessary to initialize `self.ptr` here.